fgrpc/grpcrunner.go

```diff
@@ -43,17 +43,30 @@ const (
 	prefixHTTPS      = "https://"
 )
 
-// Dial dials grpc either using insecure or using default tls setup.
-// TODO: option to specify certs.
-func Dial(serverAddr string, tls bool) (conn *grpc.ClientConn, err error) {
-	opts := grpc.WithInsecure()
-	if tls || (strings.HasPrefix(serverAddr, "https://")) {
-		opts = grpc.WithTransportCredentials(credentials.NewTLS(nil))
+// Dial dials grpc using insecure or tls transport security when serverAddr
+// has prefixHTTPS or cert is provided. If override is set to a non empty string,
+// it will override the virtual host name of authority in requests.
+func Dial(serverAddr string, cacert, override string) (conn *grpc.ClientConn, err error) {
```

Comment:
NeedFix: either put the type for all 3 or only once but here it's odd to have string 2 out of 3 args